data/movie/Nobody (2021)/2021-4k/disc01-summary.txt

```diff
@@ -82,16 +82,6 @@ Segment map: 410,411
 Type: Extra
 File name: Breaking Down the Action Tool and Die.mkv
 
-Name: Breaking Down the Action: Car Chase
```

Comment:
Why does this item need to be removed? Is it because it is a subset of another item?

Reply:
It's a duplicate of the title defined on [line 65](https://github.com/TheDiscDb/data/blob/b64fe6ebfd1c4f6b38651207aaae0e12fce5bc6f/data/movie/Nobody%20(2021)/2021-4k/disc01-summary.txt#L65).